roles/openmrs/templates/usr/share/tomcat7/.OpenMRS/openmrs-runtime.properties.j2

```diff
@@ -1,6 +1,7 @@
 connection.username={{ openmrs_mysql_user }}
 connection.password={{ openmrs_mysql_password }}
-connection.url=jdbc:mysql://{{ openmrs_mysql_host }}:{{ openmrs_mysql_port }}/openmrs?autoReconnect=true&sessionVariables=default_storage_engine=InnoDB&useUnicode=true&characterEncoding=UTF-8&zeroDateTimeBehavior=convertToNull
+connection.url=
+jdbc:mysql://{{ openmrs_mysql_connection_url }}
```

Comment:
@manutarus I don't think you want to have `jdbc:mysql://` before `{{ openmrs_mysql_connection_url }}`

Reply:
Yeah, thanks for catching that